src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -131,54 +162,71 @@ class PeerActor(nodeInfo: NodeInfo) extends Actor with ActorLogging {
         log.info("Peer probably hasn't arrived to DAO yet")
         context become new HandshakedHandler(rlpxConnection).receive
       }
+
     case DaoHeaderReceiveTimeout => disconnectFromPeer(rlpxConnection, Disconnect.Reasons.TimeoutOnReceivingAMessage)
+
+    case GetStatus => sender() ! StatusResponse(Handshaking)
+  }
+
+  private def disconnectFromPeer(rlpxConnection: RLPxConnection, reason: Int): Unit = {
+    rlpxConnection.sendMessage(Disconnect(Disconnect.Reasons.TimeoutOnReceivingAMessage))
+    system.scheduler.scheduleOnce(5.seconds, self, PoisonPill)
```

Comment:
@adamsmo maybe we can do a cleanup of this constants after https://github.com/input-output-hk/etc-client/pull/21 is merged ... I left some around too.